docs/cassandra/cassandra-test.md

```diff
@@ -15,6 +15,11 @@ to launch tests. Before running Cassandra tests, be sure, that a local
 Cassandra instance using Docker is provided
 [here](https://github.com/pomadchin/geotrellis/blob/feature/cassandra-nmr/scripts/cassandraTestDB.sh).
 
+One can also use [ccm](https://github.com/riptano/ccm) for running a Cassandra 
```

Comment:
I've tested this PR by changing the configuration in `reference.conf` and then running against a test `geotrellis` cassandra cluster built by `ccm` with vnodes enabled and partition key caching on.